examples/common/io.ads

```diff
@@ -0,0 +1,22 @@
+
+with Ada.Sequential_IO; Use Ada;
+with Ada.Strings.Unbounded; use  Ada.Strings.Unbounded;
+
+with GL.Types; use GL.Types;
+
+package IO is
```

Comment:
What is the purpose of this package? The `Read_String` procedures are never used. I also think that using an `Unbounded_String` as `out` parameter is unnecessary; you could simply make it a function and return a normal String.

Why are you using `Byte_IO` instead of `Text_IO`? What's the difference between `Read_String` and `Read_String0` (they seem to do the same)?